### DISCUSSION TITLE
Bi-weekly Community Calls Proposal

### DISCUSSION THREAD

**Discussion by vaibhawvipul** [spacewalkhq/raft-rs#30] [Ideas]
- Proposing a bi-weekly call on Wednesdays

we can discuss - 
- current status
- future work
- research papers and repositories
- any help required

**Comment by vaibhawvipul** [comment]
wdyt @dharanad @dierbei ?

**Reply by dharanad** [reply]
I think its a good idea and i am up for it.

**Reply by dierbei** [reply]
> wdyt @dharanad @dierbei ?wdyt @dharanad @dierbei ？

I'm glad to do it.

But I will have a day job and will need the evening to be available. And my English is not good, I usually use a translator.

**Comment by rishitc** [comment]
Hi @vaibhawvipul,
Was this proposal to have bi-weekly calls accepted? If yes, how can contributors join these calls?

Thanks

**Reply by vaibhawvipul** [reply]
Not accepted yet but I think I should start this for contributors. Please wait for my next message.
